internal/extsvc/types.go

```diff
@@ -408,6 +408,14 @@ func GetLimitFromConfig(kind string, config interface{}) (rlc RateLimitConfig, e
 			rlc.IsDefault = false
 		}
 		rlc.BaseURL = c.Url
+	case *schema.PerforceConnection:
+		// 2/s is the default limit we enforce
+		rlc.Limit = rate.Limit(2)
```

Comment:
The default in config is 5000 per hour, so should this not be

```suggestion
		rlc.Limit = rate.Limit(5000.0 / 3600.0)
```

Which is what we did for GitHub